problems/kattis/R2/Python/tyrado/README.md

```diff
@@ -0,0 +1,16 @@
+I fully redid R2 you can see it in r2l.py
```

Comment:
don't include this as part of the README.md, instead this would be a perfect comment on the pull request